CHANGELOG.md

```diff
@@ -45,6 +46,46 @@
 - [0.9.9 and prior](#099---20170202)
 
 
+## [2.1.1]
+
+> Released 2020/08/05
+
+### Dependencies
+
+- Bumped [lua-multipart](https://github.com/Kong/lua-multipart) to `0.5.9`.
+  [#6148](https://github.com/Kong/kong/pull/6148)
+
+### Fixes
+
+##### Core
+
+- No longer reject valid characters according to RFC 3986 in the `path` attribute of the
+  Service entity
+  [#6183](https://github.com/Kong/kong/pull/6183)
+
+##### Migrations
+
+- Fixed a problem with Cassandra migrations for 2.1.0 where empty values in some
+  fields were migrated erroneously
+  [#6171](https://github.com/Kong/kong/pull/6171)
+
+##### Admin API
+
+- Fixed the units of memory counts reported by the `/status` endpoint
```

Comment:
```suggestion
Fix issue where consumed worker memory as reported by the `kong.node.get_memory_stats()` PDK method would be incorrectly reported in kilobytes, rather than bytes, leading to inaccurate values in the `/status` Admin API endpoint (and other users of said PDK method).
```